guides/common/modules/proc_upgrading-hosts-to-next-major-rhel-release.adoc

```diff
@@ -30,6 +30,11 @@ For more information, see xref:Distributing_SSH_Keys_for_Remote_Execution_{conte
 ----
 # {foreman-installer} --enable-foreman-plugin-leapp
 ----
+. If you are using a custom job template for the Leapp preupgrade check, configure Leapp to point to your template:
```

Comment:
By doing this, you're not configuring leapp in any way. You're just moving a special marker to a different job template so that foreman knows it should do some extra post-processing

Reply:
Reworded.